ZIGSIMPlus/Views/CommandDetailSettingsViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CommandDetailSettingsViewController.swift
+//  ZIGSIMPlus
+//
+//  Created by Takayosi Amagi on 2019/06/17.
+//  Copyright © 2019 Nozomu Kuwae. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SwiftyUserDefaults
+
+public class CommandDetailSettingsViewController : UIViewController {
+
+    var presenter: CommandDetailSettingsPresenterProtocol!
+    var command: Command!
+
+    @IBOutlet weak var stackView: UIStackView!
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        presenter = CommandDetailSettingsPresenter()
+
+        let settings = presenter.getCommandDetailSettings()
+        guard let settingsForCommand = settings[command] else { return }
+
+        // Render inputs for settings
+        for setting in settingsForCommand {
+            switch setting {
+            case let data as Selector:
+                let label = UILabel()
+                label.text = data.label
+                stackView.addArrangedSubview(label)
+
+                let selector = UISegmentedControl()
+                for (i, segment) in data.segments.enumerated() {
+                    selector.insertSegment(withTitle: segment, at: i, animated: true)
+                    selector.setWidth(CGFloat(data.width / data.segments.count), forSegmentAt: i)
+                }
+                selector.selectedSegmentIndex = data.value
+                selector.addTarget(self, action: #selector(selectorAction(selector:)), for: .valueChanged)
```

Comment:
Naming "Selector" is confusing with "#selector" in Swift grammar.
How about renaming to `Chooser` or `Picker` ?

Reply:
`Picker` is confusing too because `UIPickerView` exists...
Hmm, I'll rename it back to `Segmented`.